testing/resources/selective_script_hooks_repo/hooks.yaml

```diff
@@ -0,0 +1,5 @@
+-   id: bash_hook
+    name: Bash hook
+    entry: bin/hook.sh
+    language: script
+    files: '^subdir/'
```

Comment:
yeah, I'm saying you can do that without introducing a new repository by setting `files: ...` in the config file instead of the backing repo (which would allow you to reuse the script hooks repo without creating a one-off repo)